tests/Repositories/TaskRepositoryTest.php

```diff
@@ -35,11 +40,11 @@ public function test_can_store_task_with_tags_and_assignees()
 
         $createdTask = $this->taskRepo->store($task);
 
-        $getTask = Task::with(['tags', 'taskAssignee'])->findOrFail($createdTask->id);
-        $this->assertEquals($task['title'], $getTask->title);
-        $this->assertEquals($task['tags'][0], $getTask->tags[0]->id);
+        $taskRecord = Task::with(['tags', 'taskAssignee'])->findOrFail($createdTask->id);
+        $this->assertEquals($task['title'], $taskRecord->title);
+        $this->assertEquals($task['tags'][0], $taskRecord->tags[0]->id);
 
-        $pluckAssigneeIds = $getTask->taskAssignee->pluck('id');
+        $pluckAssigneeIds = $taskRecord->taskAssignee->pluck('id');
         collect($task['assignees'])->map(function ($userId) use ($pluckAssigneeIds) {
             $this->assertContains($userId, $pluckAssigneeIds);
```

Comment:
this will never assert as you replaced task variable, change it to `$taskRecord` and do the same changes everywhere